### PR TITLE
hatsu: 0.2.2 -> 0.3.0

### PR DESCRIPTION
**Pull request by kwaa**
https://github.com/importantimport/hatsu/releases/tag/v0.3.0

Tested basic functionality of binary files via:

```bash
nix run .#hatsu -- --version
HATSU_PRIMARY_ACCOUNT="kwaa.dev" HATSU_DOMAIN="hatsu.local" nix run .#hatsu
```

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by kirillrdy** [comment]
@kwaa can you also add link to release notes to commit 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions

**Comment by kwaa** [comment]
> @kwaa can you also add link to release notes to commit

done

File: pkgs/by-name/ha/hatsu/package.nix [hunk 1]
```diff
@@ -32,6 +25,8 @@ rustPlatform.buildRustPackage rec {
   meta = {
     description = "Self-hosted and fully-automated ActivityPub bridge for static sites";
     homepage = "https://github.com/importantimport/hatsu";
+    downloadPage = "https://github.com/importantimport/hatsu/releases";
```
[the remark this answers is not in the document]
Comment (kwaa) [reply pkgs/by-name/ha/hatsu/package.nix]: > this is mean for downloads which are not easy to find on the homepage or are entirely differnet

Removed

**Comment by kwaa** [comment]
@SuperSandro2000 Can you re-review it? thx

**Review by kirillrdy — APPROVED** [review]

**Comment by kwaa** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 354408`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>hatsu</li>
  </ul>
</details>

**Comment by kirillrdy** [comment]
@kwaa thank you !

**Pull request merged**